lago/utils.py

```diff
@@ -97,13 +97,13 @@ def join_all(self, raise_exceptions=True):
 def invoke_in_parallel(func, *args_sequences):
     vt = VectorThread(func_vector(func, zip(*args_sequences)))
     vt.start_all()
-    vt.join_all()
+    return vt.join_all()
```

Comment:
I'm using it in ``virt.py`` line 217, when returning the value from the thread of each vm.
I will test it using the manual job.